src/dftbp/include/allocatablelist.fypp

```diff
@@ -0,0 +1,250 @@
+#!-------------------------------------------------------------------------------------------------!
+#!  DFTB+: general package for performing fast atomistic simulations                               !
+#!  Copyright (C) 2006 - 2021  DFTB+ developers group                                              !
+#!                                                                                                 !
+#!  See the LICENSE file for terms of usage and distribution.                                      !
+#!-------------------------------------------------------------------------------------------------!
+#!
+#:mute
+#:if not defined('_ALLOCATABLELIST_FYPP_')
+#:set _ALLOCATABLELIST_FYPP_
+
+#! Disable debugging asserts if unit testing to avoid collision with ASSERT there
```

Comment:
Probably should find a workaround for this as a separate PR, as this reduces(?) testing possibilities.

Reply:
You're absolutely right. I've fixed that now.